pacehrh/man/InitializeStochasticParameters.Rd

```diff
@@ -4,10 +4,17 @@
 \alias{InitializeStochasticParameters}
 \title{Initialize Stochastic Parameters}
 \usage{
-InitializeStochasticParameters(...)
+InitializeStochasticParameters(
+  stochasticParametersSheetName = .defaultStochasticParametersSheet,
+  changeRateLimitsSheetName = .defaultChangeRateLimitsSheet
+)
 }
 \arguments{
-\item{...}{Parameters passed through to loadStochasticParameters()}
+\item{stochasticParametersSheetName}{Name of Excel sheet holding stochastic
+parameters (mean, variance, etc) for different computations}
+
+\item{changeRateLimitsSheetName}{Name of Excel sheet holding max/min ranges
```

Comment:
I think we need to update the vignetter/pacehhrh.rmd to have section to explain the ChangeRateLimit spreadsheet based ion #119 logic as they are not very straightforward

Reply:
Moving this to issue #246.